provisioning/datastore-config-server.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/local/bin/python
```

Comment:
IMO this script should be in HiSPARC/datastore#19

This is not a script for provisioning, but a production asset.